app/models/aliquot.rb

```diff
@@ -87,7 +87,7 @@ class InsertSize < Range
           ).order('tag1s.map_id ASC, tag2s.map_id ASC')
         }
   scope :untagged, -> { where(tag_id: UNASSIGNED_TAG, tag2_id: UNASSIGNED_TAG) }
-  scope :any_tags, -> { where.not(tag_id: UNASSIGNED_TAG, tag2_id: UNASSIGNED_TAG) }
+  scope :any_tags, -> { where.not(tag_id: UNASSIGNED_TAG).or(where.not(tag2_id: UNASSIGNED_TAG)) }
 
```

Comment:
This one is actually a bug-fix.
It triggered a deprecation warning, but it turns out that the future behaviour, is the one we actually want. The previous code was finding aliquots where neither tag_id or tag2_id were -1. The new code finds any aliquots with at least one tag.